README.md

```diff
@@ -9,7 +9,7 @@ Edit the `.scss` and `.js` files in the `/src` directory (*NOT* in `/static`).
 
 ## Adding new episodes
 
-1. `docker exec irlpodcast_hugo_1 hugo new episodes/201X-XX-XX-episode-XX-episode-title-here.md`
+1. `docker exec irlpodcast_hugo_1 hugo new episodes/201X-XX-XX-SXX-EXX-episode-title-here.md`
```

Comment:
I think this needs to be changed to: `episodes/201X-XX-XX-SXXEXX-episode-title-here.md`